ChangeLog

```diff
@@ -1,3 +1,7 @@
+2015-02-23 Tamer A. Mansour (drtamermansour@gmail.com)
```

Comment:
two spaces between the date and your name and two space between your name and your email address. Also use `<` `>` angle brackets instead of parenthesis. This is an old UNIX tradition :-)